polyfills/Array/from/polyfill.js

```diff
@@ -1,45 +1,34 @@
 
 // Wrapped in IIFE to prevent leaking to global scope.
 (function () {
+	function isIterable (arraylike) {
+		switch ('function') {
+			case typeof arraylike['next']:
+			case typeof arraylike[Symbol.iterator]:
```

Comment:
We can't assume `Symbol` will exist, Firefox has versions which use `@@iterator` instead.